### PR TITLE
IS-420: Add unit test coverage for fieldItem.js

### PR DESCRIPTION
**Pull request by jeevanbhushetty**
## Issue

#420 

## Screenshot

test result

## Testing instructions

Execute `yarn test:unit` from drupal-admin-ui

### CODE REVIEW

File: src/utils/api/fieldItem.test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import { setItemById, deleteItemById, getItemsAsArray } from './fieldItem';
+
+it('should add / update item to fieldItem list.', () => {
```
Comment (dawehner) [inline src/utils/api/fieldItem.test.js]: Is there a reason why you don't use any describes in this test?

File: src/utils/api/fieldItem.test.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        },
+      ],
+    ),
+  ).toEqual([
+    { id: 'f0231265-f056-4084-b9e0-534c54eb2f64', type: 'user--user' },
+    { id: 'c6c704aa-8dd9-4e6d-9ae3-4b10714f7700', type: 'taxonomy_term--tags' },
+  ]);
+});
+
+it('should delete item from fieldItem list.', () => {
```
[the remark this answers is not in the document]
Comment (jeevanbhushetty) [reply src/utils/api/fieldItem.test.js]: Yes @dawehner you are right we should write 1 describe per test function and 1 test per expect.  Updating PR with same.

**Comment by dawehner** [comment]
I'm glad we are on the same boat. Would you be interested in helping with
some more tests?

**Comment by jeevanbhushetty** [comment]
Sure @dawehner will like to add more tests in the suite.

**Comment by jeevanbhushetty** [comment]
Updated the PR with describes but Nightwatch tests are failing, don't have permissions to re-run the build. Not sure how to move forward on this.

**Pull request merged**